src/db/wallets/createWalletDetails.ts

```diff
@@ -8,13 +9,23 @@ interface CreateWalletDetailsParams {
   address: string;
   type: WalletType;
   label?: string;
-  awsKmsKeyId?: string;
+
+  // AWS KMS
+  awsKmsKeyId?: string; // depcrecated and unused, todo: remove with next breaking change
```

Comment:
```typescript
import { STSClient, GetCallerIdentityCommand } from "@aws-sdk/client-sts";

const stsClient = new STSClient({});
const data = await stsClient.send(new GetCallerIdentityCommand({}));
const accountId = data.Account;
```

Not sure if it's worth doing this now since this PR is quite large already, unless you think it simplifies code complexity.